packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/grid.md

```diff
@@ -2,4 +2,26 @@
 id: Grid
 section: layouts
 ---
-Use a **Grid** to layout content on a fixed-size 12 column grid.
+The **grid** layout places content on a fixed 12 column grid.
+
+## Elements
+
+![Example of grid layout.](./img/grid-example.png)
+1. **Grid**: The grid container, which holds all grid items.
+1. **Grid item**: An individual section of content to be placed in the grid.
+
+## Usage
+
+Grid items can be ordered in a number of ways to create groups. The span value of all grid items must equal 12, but this can be achieved by combining different section sizes.
+
+However, by following the rule that all items should span up to 12, the grid can easily be modified and readjusted:
```

Comment:
this is small but for some reason the "however" feels wrong here, cause you already have the "but this can be achieved..." which already has the positive spin to having to equal 12.

Reply:
that's a good catch - I agree! I updated this phrasing, lmk what you think